src/main/java/backend/resource/TurboLabel.java

```diff
@@ -211,4 +217,128 @@ public int hashCode() {
     public int compareTo(TurboLabel o) {
         return actualName.compareTo(o.getActualName());
     }
+
+    /**
+     * To return the TurboLabel in labels that matches labelName
```

Comment:
Assertion error because all queries are based on name of existing labels. Technically it is impossible get a single match because the user would not be able to interact with it in the first place.